### PR TITLE
add global db connection to alfred

### PR DESCRIPTION
**Pull request by appym31**
In FRS, we will have two database instances - one will be a global db holding session info and tenant data, one will be an operations db. This PR is to create a connection to global db, to be used in alfred. For tenant data, will create a separate PR

### CODE REVIEW

**Review by pradeepvairamani — APPROVED** [review]

**Review by znewton — APPROVED** [review]

File: server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class RiddlerResources implements IResources {
 export class RiddlerResourcesFactory implements IResourcesFactory<RiddlerResources> {
     public async create(config: Provider): Promise<RiddlerResources> {
         // Database connection
-        const mongoUrl = config.get("mongo:endpoint") as string;
+        const mongoUrl = config.get("mongo:operationsDbEndpoint") as string;
```
Comment (tanviraumi) [inline server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts:44]: Riddler is supposed to use the globalDB right?

File: server/routerlicious/kubernetes/routerlicious/values.yaml [hunk 2]
```diff
@@ -68,7 +68,9 @@ zookeeper:
   url: zookeeper_url:port
 
 mongodb:
-  url: mongodb_url
+  operationsDbEndpoint: mongodb_url
```
Comment (tanviraumi) [inline server/routerlicious/kubernetes/routerlicious/values.yaml:71]: @curtisman Before this is checked in, we need to update the r11s deployment config.

**Review by tanviraumi — APPROVED** [review]
Wait for the approval from @curtisman or @sumedhb1995? Before this is checked in, r11s cluster values.yaml needs to be updated.

File: server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts (hunk 1 above)
Comment (appym31) [reply server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts:44]: Sorry just catching up to this, was OOF sick for a couple days. Yes, riddler will eventually use the globalDB - I plan to do that in a separate PR.

File: server/routerlicious/kubernetes/routerlicious/values.yaml (hunk 2 above)
Comment (appym31) [reply server/routerlicious/kubernetes/routerlicious/values.yaml:71]: Copying from offline discussion, change merged by curtis here: https://dev.azure.com/fluidframework/internal/_git/ff_internal/pullrequest/45

**Pull request merged**